diff --git a/Configuration/Entitlements/Extension-ios.entitlements b/Configuration/Entitlements/Extension-ios.entitlements
index 5f83cb6e..fb2c0f0f 100644
--- a/Configuration/Entitlements/Extension-ios.entitlements
+++ b/Configuration/Entitlements/Extension-ios.entitlements
@@ -6,11 +6,11 @@
 	<true/>
 	<key>com.apple.security.application-groups</key>
 	<array>
-		<string>group.$(BUNDLE_ID_PREFIX).homeassistant$(BUNDLE_ID_SUFFIX)</string>
+		<string>group.$(BUNDLE_ID_PREFIX).hax$(BUNDLE_ID_SUFFIX)</string>
 	</array>
 	<key>keychain-access-groups</key>
 	<array>
-		<string>$(AppIdentifierPrefix)$(BUNDLE_ID_PREFIX).HomeAssistant$(BUNDLE_ID_SUFFIX)</string>
+		<string>$(AppIdentifierPrefix)$(BUNDLE_ID_PREFIX).HAX$(BUNDLE_ID_SUFFIX)</string>
 	</array>
 </dict>
 </plist>
