diff --git a/Podfile b/Podfile
index 84c29b9e..3e7ad2a3 100644
--- a/Podfile
+++ b/Podfile
@@ -35,9 +35,9 @@ pod 'Version'
 pod 'XCGLogger'
 
 pod 'Starscream', git: 'https://github.com/zacwest/starscream', branch: 'ha-swift-api'
-pod 'HAKit', git: 'https://github.com/home-assistant/HAKit.git', branch: 'main'
-pod 'HAKit/PromiseKit', git: 'https://github.com/home-assistant/HAKit.git', branch: 'main'
-pod 'HAKit/Mocks', git: 'https://github.com/home-assistant/HAKit.git', branch: 'main'
+pod 'HAKit', git: 'https://github.com/sevimuelli/HAKit.git', branch: 'main'
+pod 'HAKit/PromiseKit', git: 'https://github.com/sevimuelli/HAKit.git', branch: 'main'
+pod 'HAKit/Mocks', git: 'https://github.com/sevimuelli/HAKit.git', branch: 'main'
 
 def test_pods
   pod 'OHHTTPStubs/Swift'
