diff --git a/Podfile b/Podfile
index 84c29b9e..3e7ad2a3 100644
--- a/Podfile
+++ b/Podfile
@@ -35,9 +35,9 @@ pod 'XCGLogger'

 # Keep Starscream reference even though HAKit already install it, because it defines our fork with the necessary fix
 pod 'Starscream', git: 'https://github.com/bgoncal/starscream', branch: 'ha-URLSession-fix'
-pod 'HAKit', git: 'https://github.com/home-assistant/HAKit.git', tag: '0.4.1'
-pod 'HAKit/PromiseKit', git: 'https://github.com/home-assistant/HAKit.git', tag: '0.4.1'
-pod 'HAKit/Mocks', git: 'https://github.com/home-assistant/HAKit.git', tag: '0.4.1'
+pod 'HAKit', git: 'https://github.com/sevimuelli/HAKit.git', branch: 'main'
+pod 'HAKit/PromiseKit', git: 'https://github.com/sevimuelli/HAKit.git', branch: 'main'
+pod 'HAKit/Mocks', git: 'https://github.com/sevimuelli/HAKit.git', branch: 'main'

 def test_pods
   pod 'OHHTTPStubs/Swift'
